src/coord/src/catalog/error.rs

```diff
@@ -34,6 +34,7 @@ pub(crate) enum ErrorKind {
     InvalidTemporarySchema,
     UnsatisfiableLoggingDependency { depender_name: String },
     Storage(rusqlite::Error),
+    AmbiguousRename(String),
```

Comment:
Can you make this more structured, to match the other variants here? Something like

```
AmbiguousRename {
    depender: String,
    dependee: String,
    message: String,
}
```

Ideally `message` would be structured too, but I'm not quite sure what the right structure would be there.